ci_example_cpp/include/ci_example_cpp/file_configuration.hpp

```diff
@@ -21,37 +21,19 @@ namespace ci_example_cpp {
      */
     File_configuration(std::string yaml_file);
 
-    /** @copy 
-     */
+    /** @copydoc Gains_configuration::get_kp() */
```

Comment:
This is awesome, I did not now `@copydoc`.

Reply:
I think I am not really using to 100% of its capability yet... for some reason I just obtain a url not the exact copy. But for now that will do^^